core/src/main/java/nu/yona/server/subscriptions/service/BuddyService.java

```diff
@@ -435,7 +436,7 @@ public void removeBuddyAfterBuddyRemovedConnection(UUID idOfRequestingUser, UUID
 				.ifPresent(b -> removeBuddy(user.getId(), b));
 	}
 
-	public void setBuddyAcceptedWithSecretUserInfo(UserDto actingUser, BuddyConnectResponseMessage connectResponseMessageEntity)
+	public void setBuddyAcceptedWithSecretUserInfo(User actingUser, BuddyConnectResponseMessage connectResponseMessageEntity)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This method has 21 lines, which is greater than the 20 lines authorized. Split it into smaller methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://10.127.10.107:9000/coding_rules#rule_key=squid%3AS138)